requirements.txt

```diff
@@ -1,4 +1,4 @@
 featuretools>=0.7.0,<0.12.0
```

Comment:
can you check if featuretools 0.7.0 works with pandas 0.25?  try checking out the tagged commit for 0.7.0 and running the unit tests

Reply:
locally worked with featuretools 0.7.0